src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -143,25 +174,19 @@ export class WorkspaceSavedObjectsClientWrapper {
       options: SavedObjectsDeleteOptions = {}
     ) => {
       const objectToDeleted = await wrapperOptions.client.get(type, id, options);
-      const workspacePermitted = await this.validateMultiWorkspacesPermissions(
-        objectToDeleted.workspaces ?? [],
-        wrapperOptions.request,
-        [WorkspacePermissionMode.LibraryWrite, WorkspacePermissionMode.Management]
-      );
-
-      if (!workspacePermitted) {
-        const objectsPermitted = await this.validateObjectsPermissions(
-          [{ type, id }],
+      if (
+        !(await this.validateWorkspacesAndSavedObjectsPermissions(
+          objectToDeleted,
           wrapperOptions.request,
+          [WorkspacePermissionMode.LibraryWrite, WorkspacePermissionMode.Management],
           [
             WorkspacePermissionMode.Management,
             WorkspacePermissionMode.LibraryWrite,
             WorkspacePermissionMode.Write,
           ]
```

Comment:
```suggestion
          [
            WorkspacePermissionMode.Read,
            WorkspacePermissionMode.Write,
          ]
```

Reply:
For the workspace meta data, we're using the `LibraryWrite`, `LibraryRead` and `Management` as object ACL permission. If we change to `Write` only, we may need to add some entry point to add the `Write permission` for the saved object permission.

Reply:
@wanglam @SuZhou-Joe It seems `Management` is `LibraryWrite` + `Write`, if that's the case, can we get rid of `Management` and use `LibraryWrite` + `Write` instead?

Reply:
@ruanyl Is that means remove the `Management` permission mode here? Since we've already have a management option in the workspace form. This management won't work fine after we remove it.

Reply:
Is that means we should pass `Write` to the API side when user checked the management permission for certain user or user group in the workspace form? Then we can remove the Management Permission mode. For now the Management permission mode is a little bit confused.